src/cgnstools/CMakeLists.txt

```diff
@@ -22,17 +24,41 @@ if (NOT APPLE)
   include(FindOpenGL)
   if (OPENGL_FOUND)
     mark_as_advanced(FORCE OPENGL_INCLUDE_PATH OPENGL_gl_LIBRARY)
+    check_include_file(GL/gl.h HAVE_GL_GL_H)
+    check_include_file(OpenGL/gl.h HAVE_OPENGL_GL_H)
+    if(HAVE_GL_GL_H)
+      add_definitions(-DHAVE_GL_GL_H)
```

Comment:
add_definition is not advisable in modern cmake. We should consider using target_compile_definition instead